libraries/WebServer/examples/WebUpdate/WebUpdate.ino

```diff
@@ -11,11 +11,16 @@
 const char *host = "esp32-webupdate";
 const char *ssid = "........";
 const char *password = "........";
+const char * authUser = "admin";
+const char * authPass = "admin";
```

Comment:
Same feedback here. Generate a random password, and serial print it. I would suggest avoiding having this hard-coded in this example